src/styles/index.scss

```diff
@@ -1,3 +1,138 @@
 body {
   margin: 0;
+
+}
+
+.header{
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+  align-items: center;
+  background-color: white;
+}
+
+.container{
+ width: 200px;
+ margin: 15px;
+ text-align: center;
+ border-radius: 20px;
+ padding-top: 16px;
+ box-shadow: 0 14px 28px rgba(0,0,0,0.25);
+
+}
+
+.container__image{
```

Comment:
pls make use of sass nesting for bem blocks in the who file

Reply:
Not fixed